nautobot/dcim/models/devices.py

```diff
@@ -916,6 +916,7 @@ def vc_interfaces(self):
             filter_q |= Q(device__virtual_chassis=self.virtual_chassis, mgmt_only=False)
         return Interface.objects.filter(filter_q)
 
+    # TODO - I am not sure if this should change with the introduction of modules
```

Comment:
Yeah probably `VirtualChassis.member_interfaces` needs to roll up from `Device.all_interfaces()` as well?